go.mod

```diff
@@ -12,4 +12,5 @@ require (
 	github.com/nicksnyder/go-i18n/v2 v2.0.3
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/sys v0.2.0 // indirect
```

Comment:
try if we can remove the indirect imports

Reply:
If we remove that circle ci fails

Reply:
if we just remove //indirect try this once 
if this works

Reply:
it gives the following warning:
`golang.org/x/sys should be indirectgo mod tidy
`